util/src/main/java/io/camunda/zeebe/util/sched/ActorControl.java

```diff
@@ -334,7 +335,7 @@ private TimerSubscription scheduleTimer(
   }
 
   /** can be called by the actor to yield the thread */
-  public void yield() {
+  public void yieldThread() {
     final ActorJob job = ensureCalledFromWithinActor("yield()");
```

Comment:
I was unsure about it. Should we also change the name on this line?

Reply:
Yes, let's avoid confusion there.